### ISSUE TITLE
Missing enum GL_VERTEX_ARRAY

### ISSUE THREAD

**Issue by bioglaze** [DerelictOrg/DerelictGL3#48]
Thanks for fixing `glObjectLabel`. I can now use it with other enums, but `GL_VERTEX_ARRAY` is not defined.

**Comment by mdparker** [comment]
Yes, it is actually. When you import the `gl3` module, you're getting all of the modern stuff and none of the deprecated stuff. To use deprecated features, like GL_VERTEX_ARRAY, replace your `gl3` import with `gl` and `DerelictGL3` with `DerelictGL`, e.g.

```d
import derelict.opengl3.gl;

void main() {
    DerelictGL.load();
    // create context
    ...
    DerelictGL.reload();
}
```
Note that this loads both the deprecated *and* modern stuff and the single import pulls everything in. A common mistake is to continue needlessly importing the gl3 module and loading DerelictGL3.

**Comment by bioglaze** [comment]
Is it really deprecated? It's listed in OpenGL 4.5 specification in table 20.4 that lists valid arguments for `glObjectLabel` and how else should one give a label to a VAO? I'm using OpenGL 4.5 Core Profile without anything deprecated.

**Comment by mdparker** [comment]
Well, I don't know anything about `glObjectLabel`. But I was somehow under the impression that `GL_VERTEX_ARRAY` was zapped along with `glEnableClientState` and friends. Apparently, I was mistaken. It's present in the latest glcorearb.h from OpenGL.org. 

I'll push out a new release when I can. In the meantime, you can define it locally with the value '0x8074' (or just import `derelict.opengl3.deprecatedConstants` to get it.

**Comment by mdparker** [comment]
Fixed it in release 1.0.21. Thanks!
